### ISSUE TITLE
For training acc_bit_mean

### ISSUE THREAD

**Issue by dreamer121121** [HiDream-ai/VAREdit#10]
> Thanks for your attention:
> 
> 1. Recommend using 16 * A800 80GB, and it's ok to use 8 * A800 (double time cost) and 32 * A800 (half time cost)
> 2. When using 16 * A800 80GB, it takes about 15-20 min / 100 iters (2.2B, 256*256 res).
> 
>  One more question: Could you please tell me what Accm bit accuracy can be achieved during training with the 2.2B model at a resolution of 256×256? The authors mention that 80% is a decent training baseline, yet my model only reaches around 68% in training.

<img width="459" height="327" alt="Image" src="https://github.com/user-attachments/assets/a4523566-d8df-4bc7-9fbf-97512462ee3d" />

 _Originally posted by @dreamer121121 in [#9](https://github.com/HiDream-ai/VAREdit/issues/9#issuecomment-4412900461)_

**Comment by Double680** [comment]
Around 80-85 during my training, maybe it depends on the data quality.

**Comment by dreamer121121** [comment]
I notice your paper mentions that training at (2.2B, 256×256) requires 8000 iterations. Combined with your statement of 20 minutes per 100 iterations, this means it only takes 26 hours of training to achieve an Acc Bit Mean of 80-85 under the (2.2B, 256×256) setting.
Is my understanding correct? Thank you very much for your reply.

**Comment by Double680** [comment]
> I notice your paper mentions that training at (2.2B, 256×256) requires 8000 iterations. Combined with your statement of 20 minutes per 100 iterations, this means it only takes 26 hours of training to achieve an Acc Bit Mean of 80-85 under the (2.2B, 256×256) setting. Is my understanding correct? Thank you very much for your reply.

During our own training, the answer is yes.

**Comment by dreamer121121** [comment]
> > I notice your paper mentions that training at (2.2B, 256×256) requires 8000 iterations. Combined with your statement of 20 minutes per 100 iterations, this means it only takes 26 hours of training to achieve an Acc Bit Mean of 80-85 under the (2.2B, 256×256) setting. Is my understanding correct? Thank you very much for your reply.
> 
> During our own training, the answer is yes.

I used about 50,000 pieces of data, 4 A100 GPUs, and a batch size (bs) of 768. Currently, I have been training for 50 hours, but the acc_bit_mean can only reach 75%. I find this a bit strange. Since our data volume is much smaller than yours, I think it should converge to 80% quickly. Below are our training configurations and training curves. Is there a problem with our training configurations? Could you give us some suggestions?

<img width="523" height="322" alt="Image" src="https://github.com/user-attachments/assets/c3aa8e9d-90af-4ddc-a011-2c7376bcd7de" />

Config: 
wandb online
pn=0.06M
exp_name=infinity_2B_data_Mini5w_pn_${pn}_bs768_fixdataloader2
bed_path=checkpoints/${exp_name}/
data_path=/root/autodl-tmp/data/mini-5w
raw_image_path=/root/autodl-tmp/data/raw-images
video_data_path=''
local_out_path=$LOCAL_OUT/${exp_name}

torchrun \
--nproc_per_node=${nproc_per_node} \
--nnodes=${nnodes} \
--node_rank=${node_rank} \
--master_addr=${master_addr} \
--master_port=${master_port} \
train.py \
--ep=10000 \
--opt=adamw \
--cum=3 \
--sche=lin0 \
--fp16=2 \
--ada=0.9_0.97 \
--tini=-1 \
--tclip=5 \
--flash=0 \
--alng=5e-06 \
--saln=1 \
--cos=1 \
--enable_checkpointing=full-block \
--local_out_path ${local_out_path} \
--task_type='t2i' \
--data_path=${data_path} \
--raw_image_path=${raw_image_path} \
--video_data_path=${video_data_path} \
--exp_name=${exp_name} \
--tblr=1e-5 \
--pn ${pn} \
--model=2bc8 \
--lbs=192 \
--workers=16 \
--short_cap_prob 0.5 \
--online_t5=1 \
--use_streaming_dataset 1 \
--iterable_data_buffersize 30000 \
--Ct5=2048 \
--t5_path=/root/autodl-tmp/pretrained/flan-t5-xl \
--vae_type 32 \
--vae_ckpt=/root/autodl-tmp/pretrained/infinity_vae_d32reg.pth  \
--wp 0.00000001 \
--wpe=1 \
--dynamic_resolution_across_gpus 1 \
--enable_dynamic_length_prompt 1 \
--reweight_loss_by_scale 1 \
--add_lvl_embeding_only_first_block 1 \
--rope2d_each_sa_layer 1 \
--rope2d_normalized_by_hw 2 \
--use_fsdp_model_ema 0 \
--always_training_scales 100 \
--use_bit_label 1 \
--zero=2 \
--save_model_iters_freq 1000000 \
--log_freq=5 \
--checkpoint_type='torch' \
--prefetch_factor=16 \
--noise_apply_strength 0.3 \
--noise_apply_layers 13 \
--apply_spatial_patchify 0 \
--use_flex_attn=False \
--pad=128 \
--auto_resume=False \
--rush_resume=/root/autodl-tmp/pretrained/infinity_2b_reg.pth
